api/src/plugins/redirect-with-message.test.ts

```diff
@@ -0,0 +1,101 @@
+import Fastify, { FastifyInstance } from 'fastify';
+import qs from 'query-string';
+
+import redirectWithMessage from './redirect-with-message';
+
+async function setupServer() {
+  const fastify = Fastify();
+  await fastify.register(redirectWithMessage);
+  return fastify;
+}
+
```

Comment:
```suggestion

      const isString = (value: unknown): value is string => {
        return typeof value === 'string';
      };
```

Reply:
I'm not sure this suggest will do that, but you're welcome to push a commit to move it around.